frame_transformer/src/auto_transition_main.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
```

Comment:
Why is this auto_transition needed? Is there a chance that both auto_transition and lifecycle_main are run together to spin different nodes? Is that the intention?

Reply:
These are two different executables. One adheres to a lifecycle approach, the other behaves more like a regular node. In carma we will be using the lifecycle_main, but I wanted this other executable to make this node more easily reusable in a debugging or data post-processing context so we don't need to call lifecycle activate each time.